process/block/metablock.go

```diff
@@ -2139,30 +2139,64 @@ func (mp *metaProcessor) verifyTotalAccumulatedFeesInEpoch(metaHdr *block.MetaBl
 	return nil
 }
 
-func (mp *metaProcessor) computeAccumulatedFeesInEpoch(metaHdr data.MetaHeaderHandler) (*big.Int, *big.Int, error) {
-	currentlyAccumulatedFeesInEpoch := big.NewInt(0)
-	currentDevFeesInEpoch := big.NewInt(0)
+func (mp *metaProcessor) getCurrentlyAccumulatedFees(metaHdr data.MetaHeaderHandler) (*big.Int, *big.Int, error) {
```

Comment:
we also need handling for the epoch change proposed block -> when executing that block, the accumulated fees for the previous epoch are used to compute the rewards and distributed or burned, and the accumulatedFees and accumulatedDevFees are reset. 

I think we miss the reset.